mage_ai/data_preparation/templates/utils.py

```diff
@@ -10,3 +12,33 @@ def copy_templates(template_path: str, dest_path: str) -> None:
     if not os.path.exists(template_path):
         raise IOError(f'Could not find templates for {template_path}.')
     shutil.copytree(template_path, dest_path)
+
+
+def build_template_from_suggestion(suggestion: Dict) -> str:
+    """
+    Creates a file template from a suggestion.
+
+    Args:
+        suggestion (Dict): Suggestion payload generated from `BaseRule`.
+
+    Returns:
+        str: String version of Python code to execute to run this cleaning suggestion.
+    """
+    clean_title = suggestion['title'].lower().replace(' ', '_')
+    cleaned_payload = json.dumps(suggestion['action_payload'], indent=4)
+    cleaned_payload = cleaned_payload.replace('\n', '\n    ')
+    template = f"""from mage_ai.data_cleaner.transformer_actions.base import BaseAction
```

Comment:
You can put the template in a file and then render the file with the suggestion input.
Maybe take a look at the jinja template

Reply:
I was exactly thinking this method and I'm currently working on a similar solution, but I didn't think to use Jinja. I'll look into using jinja then.

Reply:
Switched to Jinja for loading template